homeassistant/components/fritz/sensor.py

```diff
@@ -57,7 +59,7 @@ def _retrieve_external_ip_state(status, last_value):
 
 
 async def async_setup_entry(
-    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: Callable
```

Comment:
```suggestion
    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
```
And `from homeassistant.helpers.entity_platform import AddEntitiesCallback` is needed of course.